packages/blocks/src/attachment-block/attachment-edgeless-block.ts

```diff
@@ -0,0 +1,85 @@
+import type { HoverController } from '@blocksuite/affine-components/hover';
+
+import { AttachmentBlockStyles } from '@blocksuite/affine-model';
+import {
+  EMBED_CARD_HEIGHT,
+  EMBED_CARD_WIDTH,
+} from '@blocksuite/affine-shared/consts';
+import { toGfxBlockComponent } from '@blocksuite/block-std';
+import { customElement } from 'lit/decorators.js';
+import { styleMap } from 'lit/directives/style-map.js';
+
+import type { EdgelessRootService } from '../root-block/index.js';
+
+import { AttachmentBlockComponent } from './attachment-block.js';
+
+@customElement('affine-edgeless-attachment')
+export class AttachmentEdgelessBlockComponent extends toGfxBlockComponent(
+  AttachmentBlockComponent
+) {
+  protected override _whenHover: HoverController | null = null;
+
+  override rootServiceFlavour: string = 'affine:page';
+
+  override connectedCallback(): void {
+    super.connectedCallback();
+
+    const rootService = this.rootService;
+
+    this._disposables.add(
+      rootService.slots.elementResizeStart.on(() => {
+        this._isResizing = true;
+        this._showOverlay = true;
+      })
+    );
+
+    this._disposables.add(
+      rootService.slots.elementResizeEnd.on(() => {
+        this._isResizing = false;
+        this._showOverlay =
+          this._isResizing || this._isDragging || !this._isSelected;
+      })
+    );
+  }
+
+  override onClick(_: MouseEvent) {
+    return;
```

Comment:
Why don't just remove this?

Reply:
This is used to override the parent class's default click behaviour.